### ISSUE TITLE
bug: Api : Ban/Mute action not work

### ISSUE THREAD

**Issue by Thomasfds** [AscensionGameDev/Intersect-Engine#1449]
### Description

After starting on version 0.7.0.134, of an application that I will soon offer to the community, I updated my version to there 0.7.1.47, the actions that worked before the update (here, the ban and mute a user, according to the official api documentation) no longer works.

### Steps to Reproduce

Made api call for admin action : Ban or mute, i think other version (not writted right now in my app code) can have same issue.

### Version with bug

0.7.1.47

### Last version that worked well

0.7.0.134

### Affected platforms

Windows

### Did you find any workaround?

Nothing for now, need look source code right now for find anything can help, it's maybe just some data missed or wrong with some api change ?

### Relevant log output

```shell
à Intersect.Server.Web.RestApi.Routes.V1.UserController.DoAdminActionOnUser(Func`1 fetch, Func`1 onError, AdminActions adminAction, AdminActionParameters actionParameters)
   à Intersect.Server.Web.RestApi.Routes.V1.UserController.DoAdminActionOnPlayerById(Guid userId, String act, AdminActionParameters actionParameters)
   à lambda_method(Closure , Object , Object[] )
   à System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ActionExecutor.<>c__DisplayClass6_2.<GetExecutor>b__2(Object instance, Object[] methodParameters)
   à System.Web.Http.Controllers.ReflectedHttpActionDescriptor.ExecuteAsync(HttpControllerContext controllerContext, IDictionary`2 arguments, CancellationToken cancellationToken)
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Web.Http.Controllers.ApiControllerActionInvoker.<InvokeActionAsyncCore>d__1.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Web.Http.Controllers.ActionFilterResult.<ExecuteAsync>d__5.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Web.Http.Filters.AuthorizationFilterAttribute.<ExecuteAuthorizationFilterAsyncCore>d__3.MoveNext()
--- Fin de la trace de la pile à partir de l'emplacement précédent au niveau duquel l'exception a été levée ---
   à System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   à System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   à System.Web.Http.Dispatcher.HttpControllerDispatcher.<SendAsync>d__15.MoveNext()
```

### Duplicate Bug Check

- [X] This bug report is not a duplicate to the best of my knowledge.

**Comment by Arufonsu** [comment]
Thanks for reporting, probably related: https://github.com/AscensionGameDev/Intersect-Engine/pull/1081#issuecomment-1066025001

**Comment by Thomasfds** [comment]
Same error on kill and kick endpoint,

**Comment by lodicolo** [comment]
@Thomasfds We need the start of the error stack trace, the provided log starts in the middle so I don't know what the error is.

**Comment by Thomasfds** [comment]
> @Thomasfds We need the start of the error stack trace, the provided log starts in the middle so I don't know what the error is.

Server don't generate log, it's only the error I have posted from request response.

**Comment by lodicolo** [comment]
Will be fixed in 0.7.1.58 which I just kicked off a build for
